packages/backend/server/utils/markdown.ts

```diff
@@ -21,7 +22,7 @@ export function generateCommitPublishMessage(
         `<b>${packageName}</b>`,
         `
 \`\`\`
-npm i ${shaUrl}
+${packageManager} ${packageManager === "npm" ? "i" : "add"} ${shaUrl}
```

Comment:
let's have an object like this

```
{
npm: "i",
pnpm: "add",
yarn: "add"
```
and then based on what the package manager is, we choose!